face_recognition/api.py

```diff
@@ -200,6 +200,32 @@ def face_encodings(face_image, known_face_locations=None, num_jitters=1):
     return [np.array(face_encoder.compute_face_descriptor(face_image, raw_landmark_set, num_jitters)) for raw_landmark_set in raw_landmarks]
 
 
+def batch_face_encodings(face_images, known_face_locations=None, num_jitters=1):
+    """
+    Given an array of images, return an array where the i-th cell is an array of 128-dimension face encodings for each face in the i-th image.
+
+    :param face_images: An array of images that contains one or more faces
+    :param known_face_locations: Optional - the bounding boxes of each face in each image if you already know them.
+    :param num_jitters: How many times to re-sample the face when calculating encoding. Higher is more accurate, but slower (i.e. 100 is 100x slower)
+    :return: A list of lists of 128-dimensional face encodings (one list for each face in each image)
+    """
+    if known_face_locations is None:
+        raw_landmarks = [_raw_face_landmarks(face_image, None, model="small") for face_image in face_images]
+    else:
+        raw_landmarks = [
```

Comment:
Is this assuming that known_face_locations will be an array of arrays (one array of face locations for each image passed in, in the same order)?

That sounds reasonable to me, but I think the doc string above needs to be a little more clear about what's required since it's different than the non-batch function.